01week/resume/index.html

```diff
@@ -0,0 +1,50 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+ <title></title>
+ <link rel="stylesheet" href="./css/style.css">
+ <img src="./img/desktop-image.jpg">
```

Comment:
remember that stuff in the <head> tag doesnt show up on the page, it's just information your webpage needs (like your CSS link or meta tags). So you wanna put this <img> tag inside the <body> tag if you want it to appear on the page.